src/pages/recipient/dashboard.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
As mentioned in #71, this page should be at `/recipient`, not `/recipient/dashboard`